presto-parser/src/main/java/com/facebook/presto/sql/tree/QualifiedName.java

```diff
@@ -73,27 +82,28 @@ public String toString()
      */
     public Optional<QualifiedName> getPrefix()
     {
-        if (parts.size() == 1) {
-            return Optional.empty();
-        }
-
-        return Optional.of(new QualifiedName(parts.subList(0, parts.size() - 1)));
+        return prefix;
     }
 
-    public boolean hasSuffix(QualifiedName suffix)
+    public boolean hasSuffix(QualifiedName suffixQualifiedName)
```

Comment:
Nice, I just don't want to use a duplicate variable name like `suffix`, `name` is pretty good.